dashboard/config/bits-aux-debug.config

```diff
@@ -0,0 +1,44 @@
+### Auxiliary debug configs that detect non-critical bugs.
```

Comment:
Let's name this `bits-syzbot-debug.config` or something like this to make it consistent with `bits-syzbot.config`.

Reply:
Done